### PR TITLE
Centralize plugin management  and fix generic test checking for cropVolume availability

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

File: RegistrationLib/__init__.py [hunk 1]
```diff
@@ -3,7 +3,11 @@
...
-from LocalBRAINSFitPlugin import *
-from LocalSimpleITKPlugin import *
+
+for plugin in [
+  'Affine',
+  'ThinPlate',
+  'LocalBRAINSFit',
+  'LocalSimpleITK'
+  ]:
+  registerRegistrationPlugin('RegistrationLib.%sPlugin' % plugin)
```
Comment (pieper) [inline RegistrationLib/__init__.py]: What's the syntax for a plugin that comes from an extension?  E.g. plastimatch provides a plugin via slicerrt.

Comment (jcfr) [inline RegistrationLib/__init__.py]: Waiting I answer that question, I just did an other PR #14 including the fix for the failing generic test. Would be great to integrate it.

Comment (pieper) [inline RegistrationLib/__init__.py]: Yes, separating the two is good.  One other thing to note is that the plugin pattern is also used in the Editor and DICOM modules, so I'd like to see them all stay consistent.  I'm not opposed to a generic facility but to be honest the importlib methods look pretty clunky to my eye, while the existing pattern is a few extra lines but pretty understandable.

**Comment by jcfr** [comment]
See #15

**Pull request closed without merging**